a11y.js

```diff
@@ -1,11 +1,26 @@
+const pathOr = require('ramda/src/pathOr');
+
+global.Cypress = { env: () => {} }; // Fake Cypress.env
+const services = require('./cypress/support/config/services');
+
 const pageWidths = [320, 400, 600, 1008, 1280];
```

Comment:
this is unnecessary, can we make this an option to run on all widths and by default only run on 1 mobile width (the one that's most popular)